unreleased/mongodbreceiver-add-partial-error-handeling.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: mongodbreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Adds partial error handling with descriptive metric/attribute messages.
```

Comment:
I think we already had partial error handling, but this PR enhances the error messages.

Reply:
Updated word to enhance